### ISSUE TITLE
Express Checkout - Account Updates..??

### ISSUE THREAD

**Issue by angelleye** [angelleye/paypal-woocommerce#236]
Express Checkout obtains user details from GetExpressCheckoutDetails and uses those details to create the local user account in WordPress/WooCommerce if an account is created with the site during checkout.

I'm not sure how we're handling it, though, if that data changes next time the same user checks out.  For example, if they change their shipping address in their PayPal account, we need to make sure we update the local account accordingly when we get the next GetExpressCheckoutDetails response from their new order.

So basically, each time we get a GetExpressCheckoutDetails response we should compare that data with the local account, and if it's different, we should update the local account with the new data.

**Comment by angelleye** [comment]
This may be related to #238, #464, and/or #471
